src/main.py

```diff
@@ -185,27 +186,40 @@ def _generate_interaction_results(
     results = []
     query_cost = 0.0
```

Comment:
Our query cost seems to be summed over all requests. This is what we want, right?

Reply:
yeah